TemporaryProjects.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27703.2035
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TemporaryProjects", "TemporaryProjects\TemporaryProjects.csproj", "{36F01D35-AFD4-4144-BD1B-A01689801FFA}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TemporaryProjects", "TemporaryProjects\TemporaryProjects.csproj", "{36F01D35-AFD4-4144-BD1B-A01689801FFA}"
```

Comment:
This is expected, see https://github.com/dotnet/project-system/blob/master/docs/opening-with-new-project-system.md#project-type-guids

Reply:
Actually, I *think* I changed this manually, and it shouldn't be necessary. But I've definitely seen Visual Studio change these GUIDs automatically when migrating some of my projects and not consistently.